api/src/main/java/org/codice/ddf/admin/api/validation/SourceValidationUtils.java

```diff
@@ -41,8 +53,7 @@
         return errors;
     }
 
-    public static List<ConfigurationMessage> validateCswFactoryPid(String factoryPid,
-            String configId) {
+    public List<ConfigurationMessage> validateCswFactoryPid(String factoryPid, String configId) {
```

Comment:
I still really don't like this general source utility class knowing about _some_ of the specific sources. These validation methods better fit in their specific source configuration implementations.